src/views/app/detail-panel/edit/parts/text-editor-container.tsx

```diff
@@ -15,12 +15,27 @@ import * as StyledComp from './edit-view-styled-components';
 type PropType = {
 	onDragOverEvent: () => void;
 	draftSavedAt: string;
+	minHeight: number;
 };
-const TextEditorContainer: FC<PropType> = ({ onDragOverEvent, draftSavedAt }) => {
+const TextEditorContainer: FC<PropType> = ({ onDragOverEvent, draftSavedAt, minHeight }) => {
 	const { control, editor, throttledSaveToDraft, updateSubjectField } = useContext(EditViewContext);
 	const [Composer, composerIsAvailable] = useIntegratedComponent('composer');
 	const [t] = useTranslation();
 
+	const timeoutRef = useRef<null | ReturnType<typeof setTimeout>>(null);
+	const [showStickyTime, setStickyTime] = useState(false);
+
+	const toggleStickyTime = useCallback(() => {
+		clearTimeout(timeoutRef.current as ReturnType<typeof setTimeout>);
+		setStickyTime(false);
+		timeoutRef.current = setTimeout(() => {
+			setStickyTime(true);
+			setTimeout(() => {
```

Comment:
I'm not sure but I guess you should clear this timeout also before calling another one, but I'm not sure maybe @zovomat @nubsthead @giuliano176 can confirm it